core/controllers/home_test.py

```diff
@@ -54,11 +55,12 @@ def test_logged_in_notifications_dashboard(self):
         self.login(self.EDITOR_EMAIL)
         response = self.testapp.get('/notifications_dashboard')
         self.assertEqual(response.status_int, 200)
+        # The string I18N_TOPNAV_SIGN_IN cannot be part of the inner text of
+        # the tag, but can appear as an attribute value.
         response.mustcontain(
-            'Notifications', 'Logout',
+            'I18N_TOPNAV_NOTIFICATIONS', 'I18N_TOPNAV_LOGOUT',
             self.get_expected_logout_url('/'),
-            no=['Login', 'Your personal tutor',
-                self.get_expected_login_url('/')])
+            no=['>I18N_TOPNAV_SIGN_IN<', self.get_expected_login_url('/')])
```

Comment:
I just changed the string for the translation ID in this change, maybe it would be better to add the other test case in another pull request... this one is getting huge.